appstore/core/views.py

```diff
@@ -56,7 +56,7 @@ def get_access_token(request):
         if auth_string and ("Bearer" in auth_string):
             access_token = auth_string.split(" ")[1]
     except Exception as e:
-        logger.debug("----------> Failed getting access token. ")
+        logger.debug(f"----------> Exception: Failed getting access token. {e.__class__.__name__} ")
```

Comment:
```suggestion
        logger.debug(f"----------> {e.__class__.__name__}: Failed getting access token. {e.__class__.__name__} ")
```
This will put the exception's name at the beginning instead of a generic "Exception" first.

Reply:
What is the idiomatic way to do this? 
Coming from go, you would be generic first: then use the exception for details. Happy to do whatever is best practice for python.

Reply:
I've always used the specific exception class name to say what exception it was first, them gave more details. Just saying "Exception" isn't that helpful when almost all exception class names have the word Exception in them already.

Reply:
I committed your suggestion. I agree that looks better.